test/trento_web/controllers/cluster_controller_test.exs

```diff
@@ -1,35 +1,24 @@
 defmodule TrentoWeb.ClusterControllerTest do
   use TrentoWeb.ConnCase, async: true
 
+  import OpenApiSpex.TestAssertions
+
+  alias TrentoWeb.OpenApi.ApiSpec
+
   import Trento.Factory
 
   describe "list" do
     test "should list all clusters", %{conn: conn} do
-      [
-        %{id: cluster_id_1, name: cluster_name_1},
-        %{id: cluster_id_2, name: cluster_name_2},
-        %{id: cluster_id_3, name: cluster_name_3}
-      ] =
-        0..2
-        |> Enum.map(fn _ -> insert(:cluster) end)
-        |> Enum.sort_by(& &1.name)
+      0..2
```

Comment:
This looks really strange. Why not `insert_list`?

Reply:
Yes it is, some tests are like that, I already created a tech debt task to tackle it as the next thing in another prs